### ISSUE TITLE
propDRM improvements

### ISSUE THREAD

**Issue by vpodpecan** [vpodpecan/representation_learning#7]
In the propDRM there is plenty of code but little explanation. Could you add few more text cells which explain what is being done? Also, the output of NN training is awfully long, it would be great to shorted that as well.

**Comment by SkBlaz** [comment]
On it, yes.

**Comment by SkBlaz** [comment]
I've added more description + the molecules example. The output I would leave, as notebooks truncate it after certain length anyways ..

**Comment by vpodpecan** [comment]
You could use tqdm like:
```python
from tqdm.keras import TqdmCallback
model.fit(..., verbose=0, callbacks=[TqdmCallback(verbose=2)])
```
Besides, the propRDM notebook is a mess right now, plenty of errors. Don't you think that discretization of real valued attribues before Wordification would be beneficial? Because as it is now Wordification uses individual real values which does not make much sense.

**Comment by SkBlaz** [comment]
Errors? There seem to be only warnings. Could you be more specific?

**Comment by SkBlaz** [comment]
Btw, it seems to be tqdm_keras and not tqdm.keras

**Comment by vpodpecan** [comment]
The last cell (SHAP) is still from the trains example.

**Comment by SkBlaz** [comment]
Yes, intentionally, because the mutagenesis does not seem to work, at all.
(for this data set)

**Comment by SkBlaz** [comment]
I'll try discretizing it -> this might solve both problems (no Trains +
SHAP), otherwise this will be rather awkward and perhaps Trains shall remain

**Comment by vpodpecan** [comment]
I fixed all the errors and completed the mutagenesis example. FYI, without normalization shap fails on wordification-based input.

**Comment by SkBlaz** [comment]
Normalization of the data? I'm kind of in the middle of implementing the
discretization version, so this is kind of obsolete now it seems ..

**Comment by vpodpecan** [comment]
So I did discretization followed by wordification followed by normalization and then shap works just fine it seems.

**Comment by SkBlaz** [comment]
But this is not normalization then? I know this works fine, as I've also
done it (but not polished enough to open a pull request, was too slow
obviously), too bad then

**Comment by vpodpecan** [comment]
That was a typo, I did't do discretization twice...

**Comment by SkBlaz** [comment]
Alright, very well, so this notebook is now considered done?

**Comment by vpodpecan** [comment]
Yes.
